tests/modules/nf-core/bcftools/concat/main.nf

```diff
@@ -6,23 +6,19 @@ include { BCFTOOLS_CONCAT } from '../../../../../modules/nf-core/bcftools/concat
 
 workflow test_bcftools_concat_tbi {
     
-    input = [ [ id:'test3' ], // meta map
-              [ file(params.test_data['sarscov2']['illumina']['test_vcf_gz'], checkIfExists: true),
-                file(params.test_data['sarscov2']['illumina']['test2_vcf_gz'], checkIfExists: true) ],
-              [ file(params.test_data['sarscov2']['illumina']['test_vcf_gz_tbi'], checkIfExists: true),
-                file(params.test_data['sarscov2']['illumina']['test2_vcf_gz_tbi'], checkIfExists: true) ]
-            ]
+    input = [ 
+        [ id:'test' ], // meta map
+        [
+          file(params.test_data['homo_sapiens']['illumina']['test_haplotc_cnn_vcf_gz'], checkIfExists: true),
+          file(params.test_data['homo_sapiens']['illumina']['test_genome_vcf_gz'], checkIfExists: true) 
+        ],
+        [
+          file(params.test_data['homo_sapiens']['illumina']['test_genome_vcf_gz_tbi'], checkIfExists: true),
+          file(params.test_data['homo_sapiens']['illumina']['test_haplotc_cnn_vcf_gz_tbi'], checkIfExists: true)
+        ]
+    ]
 
-    BCFTOOLS_CONCAT ( input )
-}
+    bed = file(params.test_data['homo_sapiens']['genome']['genome_bed'], checkIfExists: true)
```

Comment:
I'm assuming you meant to add the bed int the test- but I guess there should be a test both with and without a bed?

Reply:
I'll add it, thanks!